03-discovery-specification.bs

```diff
@@ -0,0 +1,124 @@
+<pre class='metadata'>
+Title: TREE Discovery and Context Information
+Shortname: TREEDiscovery
+Level: 1
+Status: w3c/CG-DRAFT
+Markup Shorthands: markdown yes
+Group: TREE hypermedia community group
+URL: https://w3id.org/tree/specification/discovery
+Repository: https://github.com/treecg/specification
+Mailing List: public-treecg@w3.org
+Mailing List Archives: https://lists.w3.org/Archives/Public/public-treecg/
+Editor: Pieter Colpaert, https://pietercolpaert.be
+Abstract:
+    This specification defines how a client selects a specific dataset and search tree, as well as extracts relevant context information.
+</pre>
+
+# Definitions # {#overview}
+
+A `tree:Collection` is a subclass of `dcat:Dataset` ([[!vocab-dcat-3]]).
+The specialization being that this particular dataset is a collection of _members_.
+
+A `tree:SearchTree` is a subClassOf `dcat:Distribution`.
+The specialization being that it uses the main TREE specification to publish a search tree.
+
+A node from which all other nodes can be found is a `tree:RootNode`.
+
+Note: The `tree:SearchTree` and the `tree:RootNode` MAY be identified by the same IRI when no disambiguation is needed.
+
+A TREE client MUST be provided with a URL to start from, which we call the _entrypoint_.
+
+# Initializing a client with a url # {#starting-from}
+
+The goal of the client is to understand what `tree:Collection` it is using, and to find a `tree:RootNode` to start the traversal phase from.
+This discovery specification extends the initialization step in the TREE specification for the cases in which multiple options are possible.
```

Comment:
This discovery specification extends the initialization step in the TREE specification `[COMMA]` for the cases in which multiple options are possible.